x-pack/solutions/search/plugins/search_query_rules/public/components/empty_prompt/empty_prompt.tsx

```diff
@@ -240,6 +251,9 @@ export const EmptyPrompt: React.FC<EmptyPromptProps> = ({ getStartedAction }) =>
                       defaultMessage: 'Create in Console',
                     })}
                     showIcon
+                    onClick={() => {
```

Comment:
FYI: the `TryInConsoleButton` can also take a custom `data-test-subj` now that can be used for tracking click events.